templates/test/module_select.html.ep

```diff
@@ -0,0 +1,18 @@
+<select class="custom-select" name="pause-at-module" id="developer-pause-at-module">
```

Comment:
As a follow up, we can set to disabled the options that are no longer available.

Reply:
Ok, would be quite easy I guess.